Doc/library/stdtypes.rst

```diff
@@ -2950,7 +2952,8 @@ data and are closely related to string objects in a variety of other ways.
    A reverse conversion function exists to transform a bytes object into its
    hexadecimal representation.
 
-   .. method:: hex([sep[, bytes_per_sep]])
+   .. method:: hex(*, bytes_per_sep=1)
+               hex(sep, bytes_per_sep=1)
```

Comment:
Perhaps?: 
```suggestion
   .. method:: hex(sep='', bytes_per_sep=1)
```

It's sort-of accurate, by default there is no separator, but running `sep=''` will yield an error (sep must be length 1)